float64 x_dot
float64 y_dot
float64 z_dot
---
float64 success